overlay/lower/usr/share/udhcpc/default.script

```diff
@@ -73,6 +73,17 @@ case "$1" in
 			echo "Adding DNS server $i"
 			echo "nameserver $i" >>$tmpfile
 		done
+                if [ -n "$ntpsrv" ]; then
+                        echo "NTP servers provided by DHCP: $ntpsrv"
+                        # Erase /etc/ntp.conf before adding new servers
+                        > /etc/ntp.conf
+                        # Add NTP servers to /etc/ntp.conf
+                        for t in $ntpsrv; do
+                                echo "Adding NTP server $t"
+                                echo "server $t iburst" >> /etc/ntp.conf
```

Comment:
this is an issue. constant writing to the flash on reconnects will degrade flash chip.

Reply:
Should we move it to tmp? Symlink?

Reply:
Yes. We could copy the cached version to tmp and make it a working copy if nothing is sent in dhcp, or override the working copy with servers from dhcp.